judgments/utils.py

```diff
@@ -51,6 +51,14 @@ def perform_advanced_search(
     return SearchResults.create_from_string(multipart_data.parts[0].text)
 
 
+def preprocess_query(query):
+    return remove_unquoted_stop_words(normalise_quotes(query))
```

Comment:
I'd be tempted to break this down so it's more obvious what it's doing (and the order), even though it's more lines:

```suggestion
   query = normalise_quotes(query)
   query = remove_unquoted_stop_words(query)
   return query
```

Reply:
hah lol, I nearly went the other way and started using functools to compose functions together, (my Haskell past coming back to bite me) but yes, agreed :-)